src/foremast/s3/s3apps.py

```diff
@@ -189,6 +189,22 @@ def _put_bucket_encryption(self):
         LOG.debug('Response setting up S3 encryption: %s', _response)
         LOG.info('S3 encryption configuration updated')
 
+    def _put_bucket_lifecycle(self):
+        """Adds bucket lifecycle configuration."""
+        status = 'deleted'
+        if self.s3props['lifecycle']['enabled']:
+            lifecycle_config = {
+                'Rules': self.s3props['lifecycle']['lifecycle_rules']
+            }
+            LOG.debug('Lifecycle Config: %s', lifecycle_config)
+            _response = self.s3client.put_bucket_lifecycle_configuration(Bucket=self.bucket,
+                                                                         LifecycleConfiguration=lifecycle_config)
+            status = 'applied'
+        else:
+            _response = self.s3client.delete_bucket_lifecycle(Bucket=self.bucket)
+        LOG.debug('Response setting up S3 lifecycle: %s', _response)
+        LOG.info('S3 lifecycle configuration %s', status)
```

Comment:
I still want what @sijis suggested with returning anything possibly useful. Looking at `boto3` documentation, `_response` will always be `None` at the moment. In lieu of that, please return the `status` for now.

Reply:
The goal was to stylistically keep it similar to the existing source. If we want to retool returns and responses, we can. But that should be handled outside of this PR. In addition, I will be returning nothing with no use cases currently. If we had a use case sure, but today there was no use case.